scripts/checks.sh

```diff
@@ -3,8 +3,20 @@ DIAGNOSE_VERSION=4.20.13
 # Don't run anything before this source as it sets PATH here
 # shellcheck disable=SC1091
 source /etc/profile
+
+# Newer OS versions do not set REGISTRY_ENDPOINT via
```

Comment:
generally speaking we should just trust the cloud and ignore whatever we get on device, as that is the source of truth going forward.

Reply:
Good point -- I'll add that to the comment, as I think it definitely helps add context here.